DI-2086/extract_vaf_dp_filter.py

```
@@ -0,0 +1,289 @@
+        If vcf_path is None or empty
+
+    Notes
+    -----
+    Uses bcftools query to extract variant information. If the variant is not
+    found or bcftools fails, returns (0, 0.0) as default values.
+    """
+    pos = int(pos)
+    print(f"Extracting data for {chrom}:{pos} {ref} -> {alt} from VCF: {vcf_path}")
+    # use bcftools with subprocess
+    if not vcf_path:
+        raise FileNotFoundError(f"VCF file not found for {chrom}:{pos} {ref} -> {alt}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)